pkgs/shells/bash/blesh/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, stdenvNoCC, fetchFromGitHub, git, curl, jq, nix-prefetch, bashInteractive, runtimeShell }:
+
+stdenvNoCC.mkDerivation rec {
+  name = "ble.sh";
+  version = "2022-07-21";
+
+  src = fetchFromGitHub {
+    owner = "akinomyoga";
+    repo = "ble.sh";
+    rev = "a45077599d5c48d946de6e9b3c4baaaae9fc2633";
+    hash = "sha256-YUfDr3wmv7UGAoDQJBuXve0R34cQua4OyooppYXJIlU=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [ git ];
+  buildInputs = [ curl jq nix-prefetch ];
+
+  doCheck = true;
+  checkInputs = [ bashInteractive ];
+
+  installFlags = [ "INSDIR=$(out)/share" ];
+  postInstall = ''
+    mkdir -p "$out/bin"
+    cat <<SCRIPT >"$out/bin/blesh-share"
+    #!${runtimeShell}
+    # Run this script to find the ble.sh shared folder
+    # where all the shell scripts are living.
+    echo "$out/share/ble.sh"
+    SCRIPT
+    chmod +x "$out/bin/blesh-share"
```

Comment:
This can be easily replaced with ``source ~/.nix-profile/share/ble.sh``.

Reply:
Is it guaranteed to be there even when it is installed by configure.nix?
Then it's okay to remove this part and we can update the docs. Docs say that there is no way to decide `share` directory and describes a solution of `PACKAGE-share`.

Reply:
Here it is: https://nixos.org/manual/nixpkgs/stable/#sec-shell-helpers

Reply:
Seeing that section for the first time. Maybe we can do this but I wouldn't have done it.